scripts/aws/deploy.py

```diff
@@ -4,7 +4,7 @@
 
 import argparse
 import logging
-from .common.deploy import deploy
+from aws.common.deploy import deploy
```

Comment:
I looked at the pipeline output. The command run starts like `/bin/bash -c "PYTHONPATH=python-libs python3 scripts/aws/deploy.py -`. I'm pretty sure this means that the correct statement here would be `from scripts.aws.common.deploy import deploy`. Try that?

Reply:
`scripts.aws` like that doesn't pass pylint.